.travis.yml

```diff
@@ -50,6 +50,17 @@ script:
   - xdebug-enable
   - vendor/bin/phpunit --coverage-clover=build/logs/clover.xml
   - xdebug-disable
+  - if [[ "$WITH_PHAR" == "true" ]]; then composer box; fi
 
 after_success:
   - php composer/bin/test-reporter
+
+deploy:
+  provider: releases
+  api_key: $GITHUB_API_KEY
```

Comment:
Yes, I do, but to be honest, as of https://github.com/codeclimate/php-test-reporter/pull/105#issuecomment-274825375, I am not sure setting it there will work as expected, but we can try, of course!